test/certs/server.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I'd prefer not to check certificates into the repository. Would it be possible to generate these from a make target before running the tests?